ccdImports/configFiles/CCD_Probate_Backoffice/AuthorisationCaseState.json

```diff
@@ -226,4 +246,4 @@
   {"LiveFrom": "01/01/2017", "LiveTo": "01/01/2020", "CaseTypeID": "GrantOfRepresentation", "CaseStateID": "Pending", "UserRole": "caseworker-probate-pcqextractor", "CRUD": "R"},
   {"LiveFrom": "01/01/2017", "LiveTo": "01/01/2020", "CaseTypeID": "GrantOfRepresentation", "CaseStateID": "Stopped", "UserRole": "caseworker-probate-pcqextractor", "CRUD": "R"},
   {"LiveFrom": "01/01/2017", "LiveTo": "01/01/2020", "CaseTypeID": "GrantOfRepresentation", "CaseStateID": "BOGrantIssued", "UserRole": "caseworker-probate-charity", "CRUD": "R"}
-]
+]
```

Comment:
also need to add the R crud value for citizen and solicitor roles for Dormant state